.gitmodules

```diff
@@ -1,9 +1,11 @@
 [submodule "spack"]
   path = spack
-  #url = https://github.com/spack/spack
-  #branch = develop
-  url = https://github.com/jcsda/spack
-  branch = jcsda_emc_spack_stack
+  ##url = https://github.com/spack/spack
```

Comment:
Note. After https://github.com/JCSDA/spack-stack/pull/613 is merged, someone will either need to update this PR with the correct submodule pointer and revert the `.gitmodules` changes, or create a subsequent PR to repair it after merging this PR.